app/views/members/home/edit.html.haml

```diff
@@ -167,6 +167,22 @@
           = I18n.t 'members.home.edit.privacy_panel.view'
           = link_to 'https://public.svsticky.nl/privacystatement.pdf' do
             = I18n.t 'members.home.edit.privacy_panel.privacy_statement'
+      
+      - if !Settings['mailchimp.interests'].blank?
```

Comment:
Why does this PR add Mailchimp settings? This should at least belong in a different PR

Reply:
No clue how this snuck into here. I thought that I had removed all overhead from the original PR